tests/test_tr.py

```diff
@@ -0,0 +1,104 @@
+        self.assertEqual(num2words(44, False, "tr"), u"kırkdört")
+        self.assertEqual(num2words(100, False, "tr"), u"yüz")
+        self.assertEqual(num2words(101, False, "tr"), u"yüzbir")
+        self.assertEqual(num2words(103, False, "tr"), u"yüzüç")
+        self.assertEqual(num2words(110, False, "tr"), u"yüzon")
+        self.assertEqual(num2words(1000, False, "tr"), u"bin")
+        self.assertEqual(num2words(1001, False, "tr"), u"binbir")
+        self.assertEqual(num2words(10000, False, "tr"), u"onbin")
+        self.assertEqual(num2words(10100, False, "tr"), u"onbinyüz")
+        self.assertEqual(num2words(11000, False, "tr"), u"onbirbin")
+        self.assertEqual(num2words(35000, False, "tr"), u"otuzbeşbin")
+        self.assertEqual(num2words(116331, False, "tr"), u"yüzonaltıbinüçyüzotuzbir")
+        self.assertEqual(num2words(116330, False, "tr"), u"yüzonaltıbinüçyüzotuz")
+        self.assertEqual(num2words(500000, False, "tr"), u"beşyüzbin")
+        self.assertEqual(num2words(501000, False, "tr"), u"beşyüzbirbin")
+        self.assertEqual(num2words(1000111, False, "tr"), u"birmilyonyüzonbir")
+        self.assertEqual(num2words(111000111, False, "tr"), u"yüzonbirmilyonyüzonbir")
+        self.assertEqual(num2words(111001111, False, "tr"), u"yüzonbirmilyonbinyüzonbir")
+        self.assertEqual(num2words(111111111, False, "tr"), u"yüzonbirmilyonyüzonbirbinyüzonbir")
+        self.assertEqual(num2words(100001000, False, "tr"), u"yüzmilyonbin")
+        self.assertEqual(num2words(100001001, False, "tr"), u"yüzmilyonbinbir")
+        self.assertEqual(num2words(100010000, False, "tr"), u"yüzmilyononbin")
+        self.assertEqual(num2words(100010001, False, "tr"), u"yüzmilyononbinbir")
+        self.assertEqual(num2words(100011000, False, "tr"), u"yüzmilyononbirbin")
+        self.assertEqual(num2words(100011001, False, "tr"), u"yüzmilyononbirbinbir")
+        self.assertEqual(num2words(101011001, False, "tr"), u"yüzbirmilyononbirbinbir")
+        self.assertEqual(num2words(101011010, False, "tr"), u"yüzbirmilyononbirbinon")
+        self.assertEqual(num2words(1101011010, False, "tr"), u"birmilyaryüzbirmilyononbirbinon")
+        self.assertEqual(num2words(101101011010, False, "tr"), u"yüzbirmilyaryüzbirmilyononbirbinon")
+        self.assertEqual(num2words(1000000000001, False, "tr"), u"birtrilyonbir")
+        self.assertEqual(num2words(0.1, False, "tr"), u"sıfırvirgülon")
+        self.assertEqual(num2words(0.21, False, "tr"), u"sıfırvirgülyirmibir")
+        self.assertEqual(num2words(1.01, False, "tr"), u"birvirgülbir")
+        self.assertEqual(num2words(1.1, False, "tr"), u"birvirgülon")
+        self.assertEqual(num2words(1.21, False, "tr"), u"birvirgülyirmibir")
+        self.assertEqual(num2words(101101011010.02, False, "tr"), u"yüzbirmilyaryüzbirmilyononbirbinonvirgüliki")
+        self.assertEqual(num2words(101101011010.2, False, "tr"), u"yüzbirmilyaryüzbirmilyononbirbinonvirgülyirmi")
```

Comment:
I don't know any of Turkish, but it seems that Google Translator suggest that there are some missing white spaces "yüzbir milyar yüzbir milyon onbirbin on virgül yirmi"

Reply:
Sure thing. I will convert the variable names to English.
The notation and decimal reading without whites paces is the financial style which is used on invoices, etc. Addition of white spaces holds for regular reading of numbers. How would you like me to implement it?

Reply:
@toofun666 Thanks a lot for take the time to make this PR
I will trust you here. I dont speak any Turkish, so if not white spaces is more used in the financial style, we should go this way. If later any one will need to add the option for the whites spaces they could add a PR.
I have notices several method declarations commented, could you clean up this and remove not used and commented code?

Reply:
Luckily we have an authority for the Turkish language and the reference for numbers is this:
http://www.tdk.gov.tr/index.php?option=com_content&view=article&id=182:Sayilarin-Yazilisi-&catid=50:yazm-kurallar&Itemid=132
Items 2 and 3 explain our case.

Reply:
Cool! Thanks again for your time and work it's really appreciated!